packages/cxl-lumo-styles/scss/integrations/cxl-vaadin-button.scss

```diff
@@ -1,5 +1,10 @@
-// @see https://github.com/vaadin/vaadin-button/blob/master/theme/lumo/vaadin-button-styles.html
-// @see https://github.com/vaadin/vaadin-button/blob/master/src/vaadin-button.html
+/**
+ * @see
+ * https://github.com/vaadin/vaadin-button/issues/142
```

Comment:
Either indent followup lines or prefix everything.

Review JSDoc and PHPDoc standards.